general/document_splitter.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "W-dlkukAsGuQ"
+   },
+   "source": [
+    "# Overview\n",
+    "\n",
```

Comment:
Maybe the title should be "Document AI General Form Splitter" to stay consistent

Reply:
Changed to "# Document AI General Document Splitter" - "document" is twice there but that's the naming :)